resolve l2q05 e l2q07

EuJota opened the pull request:
Resolução da issue #50

EuJota committed:
resolve l2q05

matheusvictor reviewed (changes requested):
@EuJota , você precisa considerar que os valores digitados não serão iguais. Por favor, dê uma revisada ;D

EuJota committed:
resolve l2q07.py

EuJota committed:
update l2q05.py

EuJota committed:
Update l2q05.py

EuJota commented:
correção realizada, tinha entendido a principio que o usuário nao digitaria valores iguais

matheusvictor reviewed (changes requested):
@EuJota : é interessante que você coloque um laço de repetição na questão `l2q05.py` para que o programa peça ao usuário para informar um segundo número distinto do primeiro tantas vezes quantas forem necessárias.

EuJota committed:
Update l2q05.py

Adicionado loop para solicitar ao usuario entradas diferentes enquanto ele digitar valores iguais

EuJota committed:
resolve l2p06.py
